dbt_sugar/core/task/doc.py

```diff
@@ -56,10 +58,123 @@ def run(self) -> int:
                 account=dbt_credentials.get("account", ""),
                 database=dbt_credentials.get("database", "dwh"),
             ).get_columns_from_table(model, schema)
-
-        self.process_model(model, columns_sql)
+        if columns_sql:
+            self.orchestrator_document_model(model, columns_sql)
         return 0
 
+    def change_model_description(self, content: Dict[str, Any], model_name: str) -> Dict[str, Any]:
+        """Updates the model description from a model.
+
+        Args:
+            content (Dict[str, Any]): content of the schema.yml.
+            model_name (str): model name to change the description from.
+            description (str): description of the model.
+
+        Returns:
+            Dict[str, Any]: with the schema.yml updated.
```

Comment:
Hmm, I don't think that's very accurate it seems this function returns the content that will be pushed or written into the `schema.yml` file am I right?